hibernate-core/src/main/java/org/hibernate/engine/jdbc/Size.java

```diff
@@ -9,11 +9,11 @@
 import java.io.Serializable;
 
 /**
- * Models size restrictions/requirements on a column's datatype.
+ * Models size restrictions/requirements on a column's data type.
  *
- * @implNote Since we do not necessarily know the datatype up front, and therefore do not necessarily know
+ * @implNote Since we do not necessarily know the data type up front, and therefore do not necessarily know
  * whether length or precision/scale sizing is needed, we simply account for both here. Additionally, LOB
- * sizes, by standard, are allowed a "multiplier", {@code K} (Kb), {@code M} (Mb) or {@code G} (Gb).
+ * sizes, by standard, are allowed a "multiplier", i.e., {@code K} (Kb), {@code M} (Mb) or {@code G} (Gb).
```

Comment:
If you're bothered by this, I would change it to a colon. I personally think the comma was perfectly fine. Not a fan of "i.e." here.

Also this should be using the oxford comma.

```suggestion
 * sizes, by standard, are allowed a "multiplier": {@code K} (Kb), {@code M} (Mb), or {@code G} (Gb).
```

Reply:
By the way, in general I always use "for example" instead of "e.g." and "that is" instead of "i.e.". 

There's no sensible reason why these should be abbreviated, and I rarely see such abbreviations in really professionally-written text.

Also, FTR, I never _ever_ use "etc." in technical writing, which screams laziness and is very prone to ambiguity.